plugins/dev-tools/src/common_test_helpers.mocha.js

```diff
@@ -64,6 +64,27 @@ export function runTestCases(testCases, createTestCallback) {
   });
 }
 
+
+/**
+ * Captures the strings sent to console.warn() when calling a function.
+ * Copies from core.
+ * @param {function} innerFunc The function where warnings may called.
+ * @return {string[]} The warning messages (only the first arguments).
+ */
+export function captureWarnings(innerFunc) {
```

Comment:
Just for organization, I'd like you to move this new function to bottom of the file and add only one newline between functions (there's 2 newlines before this one). Otherwise, looks good.